### ISSUE TITLE
Confirm there is no security issues in sharing the artefacts

### ISSUE THREAD

**Issue by kinow** [imoutsatsos/Jenkins-JobConfigurationAnalyzer#2]
Will check this later while playing with this tool.

But from what I understood after our talk about it, a user with View/Read access to the job, would have access to the whole config.xml, and a few other things from the build.

Would be nice to confirm that there is no security issue in doing that. In other words, if there is anything that normally you would need privileges, but you are still able to circumvent Jenkins' security mechanism with this tool, then we should document it :-)

**Comment by imoutsatsos** [comment]
Good point. I expect that this is a tool of interest to admins, and perhaps job developers. As a result only a group with these permissions should be able to view/read/execute this job. I'm glad you are thinking  about these things @kinow ! Thank you!
